### PR TITLE
Update 17

### PR DESCRIPTION
**Pull request by Rainb0wKey**
Update $onInteraction + $onMessageDelete

### CODE REVIEW

File: src/callbacks/onMessageDelete.md [hunk 1]
```diff
@@ -1,16 +1,35 @@
...
+Triggered when a user deletes a message.
 
-`$onMessageDelete[channelID]` is a callback, which means it's used in the command trigger *(not the code)*. The command is ran when a user deletes a message. You must replace 'channelID' with a [valid channel ID](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) or a server variable that holds the channelID (`$getServerVar[]`).
+## Syntax 
+```
+$onMessageDelete[channel ID]
+```
 
+### Parameters
+- `channel ID`: please edit it!!!
```
Comment (MineBartekSA) [inline src/callbacks/onMessageDelete.md]:
```suggestion
- `channel ID`: The channel to which the resulting message will be sent.
```

File: src/callbacks/onInteraction.md [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
-- `$onInteraction[customID]` - Command runs upon the provided 'customID' being clicked/selected/interacted with.
+## Syntax
+```
+$onInteraction
+```
+### Supports
+- **[Buttons](../guides/buttons.md)**
+- **[Select Menu](../guides/selectmenu.md)**
+- **[Modals](../guides/modals.md)**
+
```
Comment (MineBartekSA) [inline src/callbacks/onInteraction.md:13]: Add a mention about `$customID`. It is important to properly tell users that they need to use `$customID` to differentiate between different interactions

File: src/callbacks/onInteractionComplex.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# $onInteraction[]
+Triggered upon an interaction being emitted (e.g. a button being clicked).
+
+## Syntax
+```
+$onIneraction[interaction ID]
+```
+
+### Parameters
+- `interaction ID`: please edit it!!!
```
Comment (MineBartekSA) [inline src/callbacks/onInteractionComplex.md]:
```suggestion
- `interaction ID`: The custom/menu/button/interaction ID used during the creation of buttons, menus, text fields, and other components.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/callbacks/onInteraction.md (hunk 2 above)
Comment (Rainb0wKey) [reply src/callbacks/onInteraction.md:13]: Ok

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution

**Pull request merged**